src/app/users/active-users.ts

```diff
@@ -0,0 +1,42 @@
+import { MS_PER_30_DAYs, USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD } from "@config/app"
+import { baseLogger } from "@services/logger"
+import { User } from "@services/mongoose/schema"
+
+// user is considered active if there has been one transaction of more than 1000 sats in the last 30 days
+const isUserActive = async (user: UserType /* FIXME */) => {
```

Comment:
IMO moving this out of core into app is misleading.
All the code under app should be using the domain types. If we are not ready to refactor this use-case to using domain types then it should not go under app imo but stay under core.

I don't mind the cleanup - it is an improvement to remove these functions from the mongoose object but still think this file should stay under core for the time being.